src/utf8.rs

```diff
@@ -784,6 +786,8 @@ pub fn decode_last_lossy<B: AsRef<[u8]>>(slice: B) -> (char, usize) {
 }
 
 #[inline]
+// SAFETY: the decode function relies on state bean equal to ACCEPT only
+// if cp is a valid Unicode scalar value
```

Comment:
This comment should go above the `#[inline]` attribute.

Reply:
Could you use proper English here, as much as possible? Capitalization and ending sentences with a period.

Reply:
`s/bean/being`